content/02.introduction.md

```diff
@@ -2,5 +2,94 @@
 Set introduction for paper here
 Talk about problem, goal, and significance of paper
 
-## Recent Work
-Talk about what has been done in the field in regards to text mining and knowledge base integration
+### Related Work
+
+Relationship extraction is the process of detecting and classifying semantic relationships from a collection of text.
+This process can be broken down into three different categories: (1) the use of natural language processing techniques such as manually crafted rules and identifying key text patterns for relationship extraction, (2) the use of unsupervised methods via co-occurrence scores or clustering, and (3) supervised or semi-supervised machine learning using annotated datasets for classification of documents or sentences.
+In this section, we discuss selected efforts for each type of edge that we included in this project.
+
+#### Disease-Gene Associations 
+
+Substantial effort for Disease-associates-Gene (DaG) extraction comprised of using manually crafted rules and unsupervised methods.
+One study used hand crafted rules on generated dependency trees to extract Disease-associates-Gene relationships [@tag:pkde4j].
```

Comment:
```suggestion
One study used hand crafted rules based on a sentence's grammatical structure, represented as dependency trees, to extract DaG relationships [@tag:pkde4j].
```
See what you think of this - and commit to using the short form (here DaG) once you define it. Otherwise, don't specify it.